dashboard/app/controllers/api_controller.rb

```diff
@@ -100,6 +100,9 @@ def import_clever_classroom
     course_id = params[:courseId].to_s
     course_name = params[:courseName].to_s
 
+    if course_name.length > 255
```

Comment:
Even though the ticket specifies Clever, could we hit this same issue with Google Classroom courses? I'm wondering if this validation should be moved to OmniAuthSection so that it works for any flavor of Oauth section?

Reply:
Working on doing this!